### DISCUSSION TITLE
TTS Voice Customization

### DISCUSSION THREAD

**Discussion by WULFFJ** [NVIDIA-NeMo/NeMo#3147] [Q&A] [answered]
Is one to train using a pre-trained model or go through fine tuning for this?  I am using my own voice, which is English.

**Comment by XuesongYang** [comment, marked as answer]
I would suggest trying to finetune a pre-trained English Fastpitch model followed by fine-tuning a Hifigan model as a start point. Please check our tutorial here: https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/FastPitch_Finetuning.ipynb
